pipelines/library/src/common/RepoHandler.groovy

```diff
@@ -18,57 +18,77 @@ class RepoHandler {
         this.context = context
         this.configs = configs
 
-        def getAdopt = new URL(ADOPT_DEFAULTS_FILE_URL).openConnection()
+        def getAdopt = new URL(ADOPT_JENKINS_DEFAULTS_URL).openConnection()
         this.ADOPT_DEFAULTS_JSON = new JsonSlurper().parseText(getAdopt.getInputStream().getText()) as Map
     }
 
     /*
-    Getter to retrieve user's git remote config
+    Getter to retrieve user's git ci-jenkins-pipeline remote config
     */
     public Map<String, ?> getUserRemoteConfigs() {
         return configs
     }
 
     /*
-    Getter to retrieve adopt's defaults
+    Getter to retrieve adopt's ci-jenkins-pipelines defaults
     */
     public Map<String, ?> getAdoptDefaultsJson() {
         return ADOPT_DEFAULTS_JSON
     }
 
     /*
-    Getter to retrieve user's defaults
+    Getter to retrieve user's ci-jenkins-pipelines defaults
     */
     public Map<String, ?> getUserDefaultsJson() {
         return USER_DEFAULTS_JSON
     }
 
     /*
-    Setter to retrieve and save a user defaults json inside the object
+    Setter to retrieve and save a user ci-jenkins-pipelines defaults json inside the object
     */
     public Map<String, ?> setUserDefaultsJson(String url) {
         def getUser = new URL(url).openConnection()
         this.USER_DEFAULTS_JSON = new JsonSlurper().parseText(getUser.getInputStream().getText()) as Map
     }
 
     /*
-    Changes dir to Adopt's repo
+    Changes dir to adopt's ci-jenkins-pipelines repo
     */
-    public void checkoutAdopt () {
+    public void checkoutAdoptPipelines () {
         context.checkout([$class: 'GitSCM',
-            branches: [ [ name: ADOPT_DEFAULTS_JSON["repository"]["branch"] ] ],
-            userRemoteConfigs: [ [ url: ADOPT_DEFAULTS_JSON["repository"]["url"] ] ]
+            branches: [ [ name: ADOPT_DEFAULTS_JSON["repository"]["pipeline_branch"] ] ],
+            userRemoteConfigs: [ [ url: ADOPT_DEFAULTS_JSON["repository"]["pipeline_url"] ] ]
         ])
     }
 
     /*
-    Changes dir to the user's repo
+    Changes dir to the user's ci-jenkins-pipelines repo
     */
-    public void checkoutUser () {
+    public void checkoutUserPipelines () {
         context.checkout([$class: 'GitSCM',
             branches: [ [ name: configs["branch"] ] ],
             userRemoteConfigs: [ configs["remotes"] ]
         ])
     }
 
+    /*
+    Changes dir to adopt's openjdk-build repo
+    */
+    public void checkoutAdoptBuild () {
+        context.checkout([$class: 'GitSCM',
+            branches: [ [ name: ADOPT_DEFAULTS_JSON["repository"]["build_branch"] ] ],
+            userRemoteConfigs: [ [ url: ADOPT_DEFAULTS_JSON["repository"]["build_url"] ] ]
+        ])
+    }
+
+    /*
+    Changes dir to user's openjdk-build repo
+    */
+    public void checkoutUserBuild () {
```

Comment:
TODO: Open an issue after merging to make this configurable by the constructor and class level